applications/matter_bridge/overlay-bt_max_connections_app_thread_nrf5340.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Set 10 BLE connections, as it is an upper limit supported caused by using thread and BLE.
+CONFIG_BT_MAX_CONN=10
+
+# Set buffer sizes in a consistent way with the ones used by the network core.
+#id if oncomment it there is an error that rx buffer is out of range (minimal value is 69)
+#CONFIG_BT_BUF_ACL_RX_SIZE=42
+#CONFIG_BT_BUF_ACL_TX_SIZE=42
+
+# Set MTU size to fit in the single buffer and avoid fragmentation (BUF_SIZE = MTU_SIZE + 4 B of L2CAP header).
+CONFIG_BT_L2CAP_TX_MTU=80
```

Comment:
Looking at the equation in the comment, should it be 80, considering that buffer size is 42? it means the MTU will be bigger than a buffer and will not fit in it.